en/PlanView/pattern_survey.md

```diff
@@ -3,6 +3,14 @@
 A survey allows you to create a grid flight pattern over a polygonal area. 
 You can specify an arbitrary polygon, the angle and other properties of the grid, and camera settings appropriate for creating geotagged images.
 
+> **Important** If the survey area has significant elevation variation then consider enabling [Terrain Following](#terrain).
```

Comment:
@DonLakeFlyer I reworded with suggestion first, and link to the section that explains how you do this. The link is important, the restructure is to taste - I prefer to tell people the problem and solution first if possible to catch the lazy readers.
Upshot, if you want to revert, do so, but fix up the link.